src/controllers/utils/queries.js

```diff
@@ -4,6 +4,12 @@ const fullTextSearchQuery = (keyword, isUsingMainKey) => (isUsingMainKey && !key
   ? { word: { $regex: /./ } }
   : { $text: { $search: keyword } }
 );
+
+/* Searching for words that match the keyword and also the wordClass*/
+export const searchIgboTextWithWordClass = (keyword, wordClass, isUsingMainKey) => (isUsingMainKey && !keyword
+  ? {$and: [{ word: { $regex: /./ } }, {wordClass: wordClass}]}
+  : {$and: [{$text: {$search: keyword}},{wordClass: wordClass}]}
```

Comment:
nit: preserve the space between object keys and paranthesis to make reading the object easier:

```js
  ? { $and: [{ word: { $regex: /./ } }, { wordClass: wordClass }]}
  : { $and: [{ $text: { $search: keyword } }, { wordClass: wordClass }]}
```

You can just copy and paste these lines